.project

```diff
@@ -5,6 +5,11 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
```

Comment:
Pls revert this file because it doesn't seem to have anything to do with the purpose of this PR.